cli/pkg/commands/images/images.go

```diff
@@ -206,7 +235,44 @@ func (m imagesModel) viewListModelImages() string {
 	return common.TableBaseStyle.Render(t.View()) + "\n"
 }
 
-// Cmd represents the images command
+func (m imagesModel) viewDescribeModelImages() string {
+
+	var parameters []string
+	var paramItem string
+	for key, value := range m.selectedImage.BuildParams {
+		if keySlice, ok := value.([]string); ok {
+			for index, v := range keySlice {
+				if index == 0 {
+					paramItem = common.ParamKeyStyle.Render(fmt.Sprintf("%s:", key)) + fmt.Sprint(v)
+					parameters = append(parameters, paramItem)
+				} else {
+					paramItem = common.ParamKeyStyle.Render("") + fmt.Sprint(v)
+					parameters = append(parameters, paramItem)
+				}
+			}
+
+		} else {
+			paramItem = common.ParamKeyStyle.Render(fmt.Sprintf("%s:", key)) + fmt.Sprint(value)
+			parameters = append(parameters, paramItem)
+		}
+
+	}
+
+	renderView := lipgloss.JoinVertical(
+		lipgloss.Left,
+		common.TitleStyle.Render("Image Details"),
+		common.KeyStyle.Render("Name:")+m.selectedImage.Name,
+		common.KeyStyle.Render("Description:")+m.selectedImage.Description,
+		common.KeyStyle.Render("Version:")+m.selectedImage.Version,
+		common.TitleStyle.Render("Parameters:")+"",
```

Comment:
We fetched also the `URI`, so why not displaying it?
We can also consider displaying model ID, but that is expected as an input, so a user knows that already. However, it may still be beneficial when a user will be sending the output to somebody else, e.g. for debugging or similar.

Reply:
@MarianMacik yep included URI and ID to the details